InvisibleCollectorLib/Model/Debt.cs

```diff
@@ -21,6 +21,7 @@ public class Debt : AttributesModel<Item>, IRoutableModel
         internal const string NetTotalName = "netTotal";
         internal const string PaidTotalName = "paidTotal";
         internal const string DebitTotalName = "dgereebitTotal";
+        internal const string CreditTotalName = "credit_total";
```

Comment:
@pedromendes The correct name is `creditTotal`.
```suggestion
        internal const string CreditTotalName = "creditTotal";
```